cruise-control-client/setup.py

```diff
@@ -11,7 +11,7 @@
 
 setuptools.setup(
     name='cruise-control-client',
-    version='1.0.4',
+    version='1.0.6',
```

Comment:
Were the `*.egg` files added intentionally to this patch? -- If not should we update `.gitignore` to prevent their addition by mistake?

Reply:
The `*.egg` files were added unintentionally; I will add `*.egg` to `gitignore` and remove these `*.egg` files